### PR TITLE
bpo-33283: Mention PNG as a supported format by Tcl/Tk

### PR DESCRIPTION
**Pull request by andresdelfino**
https://bugs.python.org/issue33283

### CODE REVIEW

File: Doc/library/tkinter.rst [hunk 1]
```diff
@@ -788,7 +788,8 @@ Bitmap/Pixelmap images can be created through the subclasses of
 
 * :class:`BitmapImage` can be used for X11 bitmap data.
 
-* :class:`PhotoImage` can be used for GIF and PPM/PGM color bitmaps.
+* :class:`PhotoImage` can be used for GIF and PPM/PGM bitmaps. This class can
+  also be used for PNG bitmaps starting with Tcl/Tk 8.6.
```
Comment (terryjreedy) [inline Doc/library/tkinter.rst]: I don't know why 'bitmap' is used for image files.  Compressed files are not bitmaps as I understand the term.  So I think this should say (without the markup)

class PhotoImage can be used with GIF and PPM/PGM files.  PNG files can also be used, starting with Tcl/Tk 8.6.

Or replace 'file' with 'format' or 'data'.

**Review by terryjreedy — CHANGES_REQUESTED** [review]
I highly approve of adding the information.

File: Doc/library/tkinter.rst (hunk 1 above)
Comment (andresdelfino) [reply Doc/library/tkinter.rst]: What do you think about:

class PhotoImage can be used for GIF and PPM/PGM data. PNG data is also supported, starting with Tcl/Tk 8.6.

Comment (terryjreedy) [reply Doc/library/tkinter.rst]: LGTM.  Lets see when Serhiy thinks.

Comment (serhiy-storchaka) [reply Doc/library/tkinter.rst]: I would write something like: "... for images in PPM/PGM, GIF and PNG formats.  The latter is only supported in Tk 8.6."

Comment (andresdelfino) [reply Doc/library/tkinter.rst]: I agree. I'll update the PR.

Comment (andresdelfino) [reply Doc/library/tkinter.rst]: Just a small change: "... for images in PPM/PGM, GIF and PNG formats. The latter is supported starting with Tk 8.6."

To be correct regarding the upcoming 8.7.

ok?

Comment (terryjreedy) [reply Doc/library/tkinter.rst]: LGTM even better.

**Comment by andresdelfino** [comment]
I have made the requested changes; please review again

**Review by terryjreedy — APPROVED** [review]

**Comment by serhiy-storchaka** [comment]
Do you mind to make a backport to 2.7 @andresdelfino?

**Comment by serhiy-storchaka** [comment]
You can use the cherry_picker command mentioned above: `cherry_picker 4b685bf7192fff48c8effeeae4f4d64f9420ec0f 2.7`.

It will fail due to conflicts. After resolving conflicts add modified files by `git add` and run `cherry_picker --continue`. This should open a web page for creating a new PR.

**Comment by andresdelfino** [comment]
Great! I'll fix an issue I'm having that prevents me from installing packages with pip, and I'll make the backport.

**Comment by andresdelfino** [comment]
I had problems following the cherry-picker path. Had to end the process manually, here's the PR:

https://github.com/python/cpython/pull/6542

Hope it helps.

**Pull request merged**